plugin.go

```diff
@@ -77,12 +77,12 @@ func (p *Plugin) Serve() chan error {
 			procs := value.([]*Process)
 
 			for i := 0; i < len(procs); i++ {
-				cmdStr := procs[i].service.Command
 				err := procs[i].start()
 				if err != nil {
 					errCh <- err
 					return false
 				}
+				cmdStr := procs[i].service.Command
```

Comment:
The problem of accessing the `proc[i].service.Command` is in the concurrent access to the `service` field. Thus we have to copy it into the `cmdStr` before pushing it to another goroutine. You might see that in the tests.